jsonxx.h

```diff
@@ -195,6 +195,20 @@ namespace jsonxx {
         ~Value() { reset(); }
         void reset();
 
+#ifdef JSONXX_ALLOW_INVALID_TYPES
+        template<typename T>
+        [[deprecated("this type is not natively supported by jsonxx, therefore its value will be converted to 'null'")]]
+        void import( const T& t ) {
+            reset();
+            type_ = INVALID_;
+#ifdef DEBUG
+            std::cerr << "[WARN] No JSONXX support for " << typeid(t).name() << std::endl;
+#endif
+        }
+#else
+        template<typename T> void import( const T& t ) = delete;
```

Comment:
by default, trying to serialize an invalid type is now forbidden and will result in a compilation error